src/store/sagas/sagaSingleItem.js

```diff
@@ -1,34 +1,42 @@
 import { takeEvery, select, put, call } from 'redux-saga/effects';
-import { API_SINGLE_PHONE_INFO_REQUEST, API_SINGLE_PHONE_INFO_SUCCESS, API_PHONE_INFO_ERROR } from '../types';
+import { API_SINGLE_PHONE_INFO_REQUEST, API_SINGLE_PHONE_INFO_SUCCESS, API_SINGLE_PHONE_INFO_ERROR } from '../types';
 
 import { apiDataFetch } from '../../utils/customFunc';
 import { getCurrentMatch } from '../selectors';
 import { fonoapiURL, fonoToken } from '../../utils/constants';
 
 function reqSingleItem(matchNow) {
     return apiDataFetch(fonoapiURL, fonoToken, matchNow)
-        .then(res => res.find(i => i.DeviceName === matchNow))
 }
 
+// .then(res => res.find(i => i.DeviceName === matchNow)
+
 function* fetchSingleItem() {
+
     const getMatch = yield select(getCurrentMatch);
-    try {
-        const singleItem = yield call(reqSingleItem, getMatch)
+    const apiDataResponseSingle = yield call(reqSingleItem, getMatch)
+
+    // Problems with API and i can't handle witch try/catch, responese all the time: 200 
```

Comment:
You can simulate error like
```js
if (apiDataResponseSingle.status == 'error') throw apiDataResponseSingle;
```
and it will be catched with try/catch